client/termsetting/handlers/numeric.binary.ts

```diff
@@ -59,6 +59,7 @@ export function getHandler(self) {
 				{
 					tw: { term: self.term, q: self.q },
 					filter: self.filter,
+					filter0: self.vocabApi.opts?.state?.termfilter?.filter0,
```

Comment:
@siosonel please see if okay this way

Reply:
Does not seem 100% correct.. `vocabApi.opts.state` is not reactive to when `filter0` changes, so it may give stale `filter0` unless the intent is to get the initial `opts.state`. If the intent is to get the up-to-date `state.termfilter.filter0`,, should use  `filter0: self.vocabApi.state?.termfilter?.filter0` (just be careful since this copy is mutable/not frozen for some reason - maybe should be frozen as a todo).

Reply:
thanks i've pushed this fix to master